src/main/java/io/github/dft/woocommerce/model/customerapi/CustomerWrapper.java

```diff
@@ -0,0 +1,11 @@
+package io.github.dft.woocommerce.model.customerapi;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+import java.util.ArrayList;
+
+@Data
```

Comment:
@Data is in use?

Reply:
No, i will remove it. Without @Data it is working